examples/llama2/llama2_genai.json

```diff
@@ -0,0 +1,56 @@
+{

```

Comment:
Update the llama2 readme file to cover this example.

Reply:
Done!